addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2034,7 +2034,7 @@ var SnippetsMenu = Widget.extend({
             // Note: we cannot listen to keyup in .o_default_snippet_text
             // elements via delegation because keyup only bubbles from focusable
             // elements which contenteditable are not.
-            const selection = this.ownerDocument.getSelection();
+            const selection = this.$body[0].ownerDocument.getSelection();
```

Comment:
Damn :grimacing: We have `this.ownerDocument` which is the document outside the iframe when inside the SnippetMenu class and `this.ownerDocument` which is the iframe's document when inside the `SnippetEditor` class... what a nightmare ^^ Something to change in the OWL refactoring hopefully... good catch :)

Reply:
Yep, it's tricky :(